src/modules/users-login-routes/index.js

```diff
@@ -13,13 +13,16 @@ export default class UsersLoginRoutes extends HasUserModel {
     templater.default().template(__dirname+"/../../../templates/user-forgot-email.ejs")
     templater.default().template(__dirname+"/../../../templates/user-forgot-password.ejs")
     
+    let baseUrl = application.config['users'].baseUrl
```

Comment:
Presume we're putting this config on the parent module because it's used by a bunch of these submodules.